scripts/install.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+BINARY_NAME="universal-sierra-compiler"
+LOCAL_BIN="${HOME}/.local/bin"
+
+main () {
+  check_cmd curl
+  check_cmd tar
+
+  download_and_extract_binary
+
+  add_binary_to_path
+
+  echo "${BINARY_NAME} (${release_tag}) has been installed successfully."
+}
+
+check_cmd() {
+  if ! command -v "$1" &> /dev/null; then
+      echo "$1 is not available"
```

Comment:
nit: maybe we should add something like "please install {pkg} and run the script again"?